src/java.desktop/macosx/classes/com/apple/laf/AquaComboBoxUI.java

```diff
@@ -454,6 +454,32 @@ protected LayoutManager createLayoutManager() {
     }
 
     class AquaComboBoxLayoutManager extends BasicComboBoxUI.ComboBoxLayoutManager {
+        protected Rectangle rectangleForCurrentValue() {
+            System.out.println("rectForCurrentValue NEW: " + comboBox.getHeight());
```

Comment:
Missed removing print statement :)

Reply:
Fixed. Thanks for catching that. I had a lot of comments to clean up after testing and this one slipped through.